datadog-sidecar-ffi/src/lib.rs

```diff
@@ -644,6 +648,119 @@ impl<'a> TryInto<SerializedTracerHeaderTags> for &'a TracerHeaderTags<'a> {
     }
 }
 
+/// Enqueues a telemetry log action to be processed internally.
+/// Non-blocking. Logs might be dropped if the internal queue is full.
+///
+/// # Safety
+/// Pointers must be valid, strings must be null-terminated if not null.
+#[no_mangle]
+pub unsafe extern "C" fn ddog_sidecar_enqueue_telemetry_log(
+    session_id_ffi: CharSlice,
+    runtime_id_ffi: CharSlice,
+    queue_id: u64,
+    identifier_ffi: CharSlice,
+    level: ddtelemetry::data::LogLevel,
+    message_ffi: CharSlice,
+    stack_trace_ffi: Option<NonNull<CharSlice>>,
+    tags_ffi: Option<NonNull<CharSlice>>,
+    is_sensitive: bool,
+) -> MaybeError {
+    match ddog_sidecar_enqueue_telemetry_log_impl(
+        session_id_ffi,
+        runtime_id_ffi,
+        queue_id,
+        identifier_ffi,
+        level,
+        message_ffi,
+        stack_trace_ffi,
+        tags_ffi,
+        is_sensitive,
+    ) {
+        Ok(_) => MaybeError::None,
+        Err(e) => MaybeError::Some(e.into()),
+    }
```

Comment:
```suggestion
    try_c!(ddog_sidecar_enqueue_telemetry_log_impl(
        session_id_ffi,
        runtime_id_ffi,
        queue_id,
        identifier_ffi,
        level,
        message_ffi,
        stack_trace_ffi,
        tags_ffi,
        is_sensitive,
    ));

    MaybeError::None,
```
consistent with the rest of the file